tests/unit/test_lyrics.py

```
@@ -315,3 +315,155 @@ def test_backup_existing_outputs_no_wav_files(self, basic_karaoke_gen, temp_dir)
+             patch('os.path.exists', return_value=False), \
+             patch('os.makedirs'), \
+             patch('shutil.copy2'), \
+             patch('karaoke_gen.lyrics_processor.load_dotenv'), \
+             patch('karaoke_gen.lyrics_processor.LyricsTranscriber') as mock_transcriber:
+            
+            mock_instance = MagicMock()
+            mock_instance.process.return_value = mock_result
+            mock_transcriber.return_value = mock_instance
+            
+            # Should NOT raise - skip_transcription bypasses validation
+            result = basic_karaoke_gen.lyrics_processor.transcribe_lyrics(
```

Ruff (0.14.10)

Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)